samples/snippets/csharp/programming-guide/classes-and-structs/object-collection-initializers/BasicObjectInitializers.cs

```diff
@@ -114,6 +121,29 @@ public static void ObjectInitializerExamples()
             };
             // </SnippetDictionaryAddInitializer>
 
+            {
+                // <SnippetReadOnlyPropertyCollectionInitializer>
+                CatOwner owner = new CatOwner
+                {
+                    Cats =
+                    {
+                        new Cat{ Name = "Sylvester", Age=8 },
+                        new Cat{ Name = "Whiskers", Age=2 },
+                        new Cat{ Name = "Sasha", Age=14 }
+                    }
+                };
+                // </SnippetReadOnlyPropertyCollectionInitializer>
+            }
+
+            {
+                // <SnippetReadOnlyPropertyCollectionInitializerTranslation>
+                CatOwner owner = new CatOwner();
+                owner.Cats.Add(new Cat{ Name = "Sylvester", Age=8 });
+                owner.Cats.Add(new Cat{ Name = "Whiskers", Age=2 });
+                owner.Cats.Add(new Cat{ Name = "Sasha", Age=14 });
+                // <SnippetReadOnlyPropertyCollectionInitializerTranslation>
```

Comment:
@BillWagner I think there is an error that slipped here because this last snippet in [the section added by this PR](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers#object-initializers-with-collection-read-only-property-initialization) is appearing blank. The forward slash that designates it as an _end tag_, as in `</…>`, is missing. I'll submit a fix through another PR.

Reply:
I just submitted the fix via PR #24653.